### DISCUSSION TITLE
How do I know something is happening? / Is SophiApp stuck?

### DISCUSSION THREAD

**Discussion by qilinmoe** [Sophia-Community/SophiApp#208] [Q&A] [answered]
I come from a place where most of the customization tools I use are CLI, so I'm not exactly used to using GUI applications, especially ones that do not provide a "terminal" so I know what and if something is happening.

Currently I have around 50 tweaks "applying" (including the removal of some UWP apps) but I'm not sure if they are actually being applied or if the program is just doing nothing. Looking at task manager I see little to no cpu usage (1-2% max) and no disk usage.
I am not sure if SophiApp provides a terminal or logs or something that I could look at to make sure things are happening, if so could someone point me in the right direction?
(Thinking back now I should've probably used the script ^^,)

Thank you in advance :)

**Comment by farag2** [comment, marked as answer]
Hello. SophiApp provides logs but i do not recommend use it in it's current status. The development of 1.x version is over. As I stated in the Readme file SophiApp 2.0 is in active development. Better use Sophia Script. https://github.com/farag2/Sophia-Script-for-Windows
